packages/lending/src/protocol.ts

```diff
@@ -53,9 +53,9 @@ export abstract class Protocol extends common.Web3Toolkit {
     return true;
   }
 
-  abstract toUnderlyingToken(marketId: string, protocolToken: common.Token): common.Token;
+  abstract toUnderlyingToken(marketId: string, protocolToken: common.Token): common.Token | undefined;
 
-  abstract toProtocolToken(marketId: string, underlyingToken: common.Token): common.Token;
+  abstract toProtocolToken(marketId: string, underlyingToken: common.Token): common.Token | undefined;
```

Comment:
Is it better to remove `abstract` and return `undefined` in `protocol.ts` directly?